Update deprecation of commands (#1254)
demisto/content-docs: content-repo/extra-docs/articles/deprecation-process-and-hidden-packs.md

Example: Use other integration example:
<img src="../../../docs/doc_imgs/integrations/deprecated-integration-2.png" width="400" align="middle"></img>

## How to Deprecate an Integration Command
1) Add the following key to the command in the integration yaml file: 
   ```deprecated: true```
​
   **Why deprecate commands instead of deleting them?**
   Some users may have used this command in a custom script or playbook.

>    Deleting a command may result in unexpected errors, while deprecating retains backwards compatibility.

Reviewer: ```suggestion
   Deleting a command may result in unexpected errors, while deprecation retains backwards compatibility for the command.
```

>    Deleting it will result in an unexpected error, that's why deprecating is the preferred method to retain backwards compatible.

Reviewer: ```suggestion
   Deleting a command may result in unexpected errors, while deprecating them retains backwards compatibility.
```
